EDI/Operation/07_PrepareWorkflow.md

```diff
@@ -0,0 +1,78 @@
+[!!Manage workflows](../../ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+# Prepare workflow
+
+Extend your workflows by adding an action to export the EDIFACT messages. To do this, use a workflow that processes the business document you want to transfer to your EDIFACT message receiver, for example an invoice or a delivery note. 
+
+To integrate the export to the workflow, you need to insert an
+*Execute PHP code* core action. For detailed information on this core action, see [Execute PHP code](../../ActindoWorkFlow/UserInterface/08_CoreActions.md#execute-php-code) in the *Process Orchestration* docmentation. You can insert this action at any position in the workflow in which you have access to the required business document data. Bear in mind that if any errors that occur after the EDIFACT export and before posting for subsequent processes can lead to data inconsistencies that need to be cleaned up.
+
+The following example shows how to export a delivery note via an export definition.
+
+
+#### Prerequisites
+
+- You are familiar with the configuration of workflows. For detailed information, see [Manage the workflows](../../ActindoWorkFlow/Operation/01_ManageWorkflows.md) in the *Process Orchestration* documentation. 
+- The export definition is activated, see [Activate definition](01_ManageDefinitions.md#activate-definition).
+- Your workflow processes the business document (for example, the invoice) for which you have created the EDIFACT export definition. 
+
+
+#### Procedure
+
+*Workflow > Select workflow > Select version*
+
+![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Select a place with which the entity is provided that you want to export with the EDIFACT message.
+
+2. Add an *Execute PHP code* core action.
+
+    ![Add Execute PHP code](../../Assets/Screenshots/EDI/Operation/WorkflowExecutePHPCode.png "[Add Execute PHP code]")
+
+3. Enter a label in the *Label* field with which you can easily identify the purpose of this action, for example **Export to Customer 2**.
+
+4. Click the *PHP code* field in the *Configuration* section of the action settings.   
+    A pop-up window for editing the PHP code is displayed.
+
```

Comment:
Kann man einen kurzen Einführungssatz zum Screenshot hinzufügen, als Erklärung?
Vorschlag: 
5. Insert the following lines of code (example).
  [code block]
  
  Explanation 
  [table]
  
  If correctly formatted. the code is displayed as follows: 
  
    [Screenshot]

Reply:
Done